common/InternalRequestsCleaner.js

```diff
@@ -0,0 +1,21 @@
+const RequestsCleaner = require('./RequestsCleaner');
+
+class InternalRequestsCleaner extends RequestsCleaner {
+	verifySignatures() {
+		if (!this.calls) {
+			return;
+		}
+		Object.keys(this.calls).forEach((socketId) => {
+			Object.keys(this.calls[socketId]).forEach((procedure) => {
+				Object.keys(this.calls[socketId][procedure]).forEach((signature) => {
+					if (RequestsCleaner.isOutdated(signature, this.timeoutMs)) {
+						this.calls[socketId][procedure][signature]('RPC response timeout exceeded');
+						delete this.calls[socketId][procedure][signature];
```

Comment:
Some of this deep property access can be avoided by using variables. E.g.
```js
(socketId) => {
    const call = this.calls[socketId];
    Object.keys(call).forEach(...);
}
```
(Also in other files.)